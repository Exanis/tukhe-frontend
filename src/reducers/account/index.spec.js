import slice from './index';

it('Should update state with action', () => {
    const state = {};
    const action = {
        type: slice.actions.setAccounts,
        payload: 'test'
    };

    expect(slice.reducer(state, action)).toMatchSnapshot();
});
