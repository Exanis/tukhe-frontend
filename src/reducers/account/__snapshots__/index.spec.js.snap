// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should update state with action 1`] = `
Object {
  "accounts": "test",
}
`;
